test/integration/targets/nxos_install_os/tasks/upgrade/clear_persistent_sockets.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
@trishnaguha I added this because as you know this is what we have to do for tasks to execute successfully following an upgrade.  It's commented out in the actual call flow (see `install_os.yaml` below) but I decided to include it for users that might make use of it.